workstreams/supply-chain-maturity/README.md

```diff
@@ -0,0 +1,44 @@
+# Supply Chain Maturity Workstream README
+
+This is a workstream within the CDF SIG Software Supply Chain.
+
+The forming of this workstream was suggested at a [recent SIG Software Supply
+Chain
+meeting](https://github.com/cdfoundation/sig-software-supply-chain/blob/main/docs/meetings.md#August-25-2022)
+
+## Scope
+
+The workstream will cover CICD beyond build and deploy.
+
+## Mission
+
+The workstream seeks to define industry standards for Supply Chain Maturity that
+augment [SLSA](http://slsa.dev).  Where SLSA covers the supply chain from code
+through artifact, maturity covers the artifact lifecycle, including deployments,
+rollouts, testing, rollbacks, and more.
+
+The workstream seeks to define an industry standard Code Health Project Score
+("CHiPS") that augment [SLSA](http://slsa.dev).
+
+## Members
+Current members:
+* David Bendory, Google
+* Justin Abrahams, eBay
+* Ankit _name_, _company_
+* Parth Patel, _company_
+
+Membership to this workstream is open and self-declared.
+
+New members are advised to:
+* Join the [#wg-supply-chain-maturity on CDF
+  Slack](https://cdeliveryfdn.slack.com/archives/C03V8R6C64C) and introduce
+  yourself.
+* TODO: add meeting notes, agendas, etc.
+* Regularly join the workstream meetings.
+
+## Communication Channels
+There is a [Slack channel](https://cdeliveryfdn.slack.com/archives/C03V8R6C64C)
+
+## Meetings
+
+TBD
```

Comment:
Created a HackMD document for you: https://hackmd.io/xq6lH4F7RUWmqZhluMcgLw

If you pass me your HackMD username, I can invite you to CDF Team on HackMD.

Reply:
https://hackmd.io/@bendory, and thanks!

Reply:
You are added to the team on HackMD: https://hackmd.io/team/cdfoundation?nav=overview